src/app/courses/add-courses/courses-add.component.ts

```diff
@@ -49,6 +50,9 @@ export class CoursesAddComponent implements OnInit, OnDestroy {
   subjectLevels = constants.subjectLevels;
   images: any[] = [];
 
+  // from the languages import
+  languageNames = languages.map(list => this.languageNames = list.name);
```

Comment:
You should remove `this.languageNames = `.  This still has the expected result , but the return statement in the `map` should just be `list.name` not a different assignment to `this.languageNames`.